setup.py

```diff
@@ -21,9 +21,12 @@
 
 
 def main():
-    for nexus_definition_dir in ("/./", "/base_classes/", "/applications/", "/contributed_definitions/"):
-        f = open(os.path.abspath(os.path.dirname(__file__)) + "/nexusparser/definitions/" + nexus_definition_dir + "__init__.py", "a")
-        f.close()
+    try:
+        for nexus_definition_dir in ("/./", "/base_classes/", "/applications/", "/contributed_definitions/"):
+            f = open(os.path.abspath(os.path.dirname(__file__)) + "/nexusparser/definitions/" + nexus_definition_dir + "__init__.py", "a")
+            f.close()
+    except FileNotFoundError:
+        pass
```

Comment:
Is this a hack to get the .nxdl files somehow installed? There must be a better way. The manifest.in should do the job, right?

Reply:
I actually didn't add this and just pulled it from the master branch. This is not needed anymore as we have the git module now. This was hacked in previously to populate the needed files from the Nexus repo. I'll do these changes appropriately in another merge request maybe. Because this can affect other parts of the code as well. We are already working on arranging the code that primarily uses this properly in the read_nexus branch. So I'll bring in the changes here once we're done with that.